lib/tpm2.h

```diff
@@ -190,6 +194,13 @@ tool_rc tpm2_rsa_decrypt(ESYS_CONTEXT *esys_context, tpm2_loaded_object *keyobj,
         const TPMT_RSA_DECRYPT *in_scheme, const TPM2B_DATA *label,
         TPM2B_PUBLIC_KEY_RSA **message);
 
+tool_rc tpm2_rsa_encrypt(ESYS_CONTEXT *ectx,
```

Comment:
every function argument needn't be on the newline, collapse this to character width per coding guideline.